From c2e3e432e2aad8b5687c0aa7b55b078d934e0af3 Mon Sep 17 00:00:00 2001
From: Mahesh Mahadevan <r9aadq@freescale.com>
Date: Fri, 21 Oct 2011 08:44:37 -0500
Subject: [PATCH 0475/2463] ENGR00160611 Update ALSA driver to initialize
 sooner

All ALSA sound-cards are not registered, hence change the init to regular
module_init.

Signed-off-by: Mahesh Mahadevan <r9aadq@freescale.com>
---
 sound/soc/imx/imx-sgtl5000.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/sound/soc/imx/imx-sgtl5000.c b/sound/soc/imx/imx-sgtl5000.c
index 3fd0aa2..b59abf2 100644
--- a/sound/soc/imx/imx-sgtl5000.c
+++ b/sound/soc/imx/imx-sgtl5000.c
@@ -390,7 +390,7 @@ static void __exit imx_sgtl5000_exit(void)
 	platform_device_unregister(imx_sgtl5000_snd_device);
 }
 
-late_initcall(imx_sgtl5000_init);
+module_init(imx_sgtl5000_init);
 module_exit(imx_sgtl5000_exit);
 
 MODULE_AUTHOR("Sascha Hauer <s.hauer@pengutronix.de>");
-- 
1.7.7.4

